src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -72,7 +77,9 @@ internal interface ISyntaxFactsService : ILanguageService
 
         bool IsRightSideOfQualifiedName(SyntaxNode node);
 
-        bool IsMemberAccessExpressionName(SyntaxNode node);
```

Comment:
The problem with "IsMemberAccessExpressionName" is then you need "IsMemberAccessExpressionExpression".  I think that reads terrible.  So instead i have "IsNameOfMemberAccessExpression" and "IsExpressionOfMemberAccessExpression"